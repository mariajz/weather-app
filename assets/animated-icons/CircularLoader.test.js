import { render } from '@testing-library/react-native';
import React from 'react';
import CircularLoader from './CircularLoader';

describe('CircularLoader', () => {
    it('should render CircularLoader svg', () => {
        const container = render(<CircularLoader />);

        expect(container).toMatchSnapshot();
    });
});
