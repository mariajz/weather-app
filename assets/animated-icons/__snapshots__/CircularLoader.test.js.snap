// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularLoader should render CircularLoader svg 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="85.333"
  bbWidth="85.333"
  focusable={false}
  height={85.333}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 85,
        "width": 85,
      },
    ]
  }
  vbHeight={64}
  vbWidth={64}
  width={85.333}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGPath
      d="M17.7 7.7C16.9 8.4 22 18 23.2 18c1.3 0 1-1.8-1.2-6-2.1-4.1-3.3-5.3-4.3-4.3zM7.6 17.9c-.7 1 .7 2.2 5.1 4.5 3.5 1.8 5.3 2 5.3.8 0-1.4-9.8-6.3-10.4-5.3zM4 32c0 .5 2.5 1 5.5 1s5.5-.5 5.5-1c0-.6-2.5-1-5.5-1S4 31.4 4 32zM13.5 41.2c-4.9 2.6-6.5 4-6 4.9.7 1 10.5-3.9 10.5-5.3 0-1.1-2.1-1-4.5.4zM47 40.9c0 1.1 6.2 5.1 7.9 5.1 2.2 0 1-1.5-2.9-3.8-4.3-2.4-5-2.6-5-1.3zM19.6 50.9c-1.4 2.7-2.2 5.2-1.7 5.5 1 .7 2.2-.7 4.5-5.1 3.5-7 1-7.3-2.8-.4zM40 47.1c0 2.1 4.4 8.9 5.7 8.9 1 0 .7-1.4-1.3-5-2.6-4.7-4.4-6.3-4.4-3.9zM31 54.5c0 3 .5 5.5 1 5.5.6 0 1-2.5 1-5.5s-.4-5.5-1-5.5c-.5 0-1 2.5-1 5.5z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
